docs/content/mobile/ios/local-deploy.md

```diff
@@ -1,63 +1,77 @@
+# Connect to a local deployment
 
 Before reading this document, make sure you are familiar with:
 
   * [Setting up iOS Support for the GDK]({{urlRoot}}/content/mobile/ios/setup)
   * [Ways to test your iOS client]({{urlRoot}}/content/mobile/ios/ways-to-test)
 
-## Connecting your iOS device to a local deployment using Unity Remote
-You need the Unity Remote app for this. See the [Unity documentation](https://docs.unity3d.com/Manual/UnityRemote5.html) for details.
+## Connecting to a local deployment in the Editor{#in-editor}
+1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
 
-  1. Connect the mobile device to your computer using a USB cable.
-  1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
-  1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
-  1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any iOS Device**.
-  1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
-  1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
+    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+1. In your Unity Editor, open the `FPS-Development` Scene and click the Play button.<br/>
+
+## Connecting to a local deployment using Unity Remote{#unity-remote}
+You need the Unity Remote app installed on your iOS device in order to follow the below steps. See the [Unity documentation](https://docs.unity3d.com/Manual/UnityRemote5.html) for details.
+
+1. Open your project in the Unity Editor.
+1. Build your workers from the SpatialOS menu by selecting **Build for local** > **All workers**.
+1. Connect your iOS device to your computer using a USB cable. Accept the **Trust This Computer** alert if it appears.
+1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
+1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any iOS Device**.
+1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
+1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
 
     > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
 
-  1. In the Editor’s Game view, select **Play**.
+1. Open the scene that starts both your [client]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker). In the FPS Starter Project this is `FPS-Development.scene`.
```

Comment:
Using `client` and `server-worker` feels a bit inconsistent. Should we use `client-worker` and `server-worker` instead?

Reply:
+1 "Client" means your client executable.

Reply:
fixed in both iOS & Android, good spot